eadf/importers.py

```diff
@@ -186,3 +189,53 @@ def fromAngleListData(
     )
 
     return EADF(arrInter, arrAzi, arrCoEle, arrFreqData, arrPos)
+
+def loadData(
+    path: str,
+    numPor = 0,
+) -> EADF:
+    """
+    This format defines angular data over uniformly sampled
+    ele, azi, elem, pol, and freq. Using the importer it is possible to choose
+    the number of ports to derive the EADF by supplying the numPor value.
+
+    Parameters
+    ----------
+    path : string
+        Location of .mat file containing measurement data
+    numPor : int
+        Number of antenna ports to derive the EADF > 0
+
+    Returns
+    -------
+    EADF
+        Created Array
+
+    """
+
+    data = scipy.io.loadmat(path)
+
+    # this is just stuff brought to you by matlab
+    arrCoEle = (
+        np.radians(data["pattern"][0][0][0][0][0][1].flatten()) + np.pi / 2
+    )
+    arrAzi = np.radians(data["pattern"][0][0][0][0][1][1].flatten())
+    arrFreq = data["pattern"][0][0][0][0][4][1]
+    arrPattern = data["pattern"][0][0][1]
+    numPorData = (data["pattern"][0][0][0][0][2][1]).size
+
+    if (numPor > numPorData) or (numPor < 1):
+        numPor = 9
```

Comment:
what? this looks wrong. either throw an error if the function parameters are invalid, or at least issue a warning, that we did something different than the user specified.

Reply:
that is certainly incorrect.